.github/workflows/deployment.yml

```diff
@@ -26,7 +26,7 @@ env:
   IEEEAPIKey: ${{ secrets.IEEEAPIKey }}
   BiodiversityHeritageApiKey: ${{ secrets.BiodiversityHeritageApiKey}}
   OSXCERT: ${{ secrets.OSX_SIGNING_CERT }}
-  GRADLE_OPTS: -Xmx4g -Dorg.gradle.vfs.watch=false
+  GRADLE_OPTS: -Xmx4g -Dorg.gradle.daemon=false -Dorg.gradle.vfs.watch=false
```

Comment:
Why disable gradle deamon?

Reply:
It was diabled in one of the two deployments.

gradle started once - and then the 3 seconds of daemon-start-time are not needed

---

I thought, disabling the daemon would solve the windows build issue - but does not...